### PR TITLE
feature/PLAT-2546

### PR DESCRIPTION
**Pull request by ivanovSPvirtru**
Sunseting nodeJS

### CODE REVIEW

File: lib/package.json [hunk 1]
```diff
@@ -39,45 +33,39 @@
...
   },
   "scripts": {
-    "build": "npm run clean && tsc -p tsconfig-node-esm.json && tsc",
+    "build": "npm run clean && tsc",
     "clean": "rm -rf {build,coverage,dist}",
     "coverage:merge": "for x in mocha wtr; do cp coverage/$x/coverage-final.json coverage/$x.json; done; nyc report --reporter text -t coverage --lines 75 --statements 75 --branches 70 --functions 65 --check-coverage >coverage/coverage.txt",
     "doc": "typedoc --out dist/docs src/index.ts",
     "format": "prettier --write \"{src,tdf3,tests}/**/*.ts\"",
-    "license-check": "license-checker --production --onlyAllow 'Apache-2.0; BSD; CC-BY-4.0; ISC; MIT'",
+    "license-check": "license-checker-rseidelsohn --production --onlyAllow 'Apache-2.0; BSD; CC-BY-4.0; ISC; MIT'",
```
Comment (pflynn-virtru) [inline lib/package.json:46]: a forked, maintained license checker

File: lib/tdf3/src/models/attribute-set.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { Validator } from 'jsonschema';
+import Ajv from 'ajv';
```
Comment (pflynn-virtru) [inline lib/tdf3/src/models/attribute-set.ts:1]: I like `ajv`, but what was the reasoning to switch?

File: lib/tdf3/src/version.ts [hunk 3]
```diff
@@ -1,4 +1,2 @@
-import { inBrowser } from './utils/index.js';
-
 export const version = '1.0.0';
```
Comment (pflynn-virtru) [inline lib/tdf3/src/version.ts:1]: what is this version?  the TDF3 schema version?

Comment (pflynn-virtru) [reply lib/tdf3/src/version.ts:1]: or package version?   can we bump the package version to 2.0.0

**Review by pflynn-virtru — APPROVED** [review]

File: lib/tdf3/src/models/attribute-set.ts (hunk 2 above)
Comment (ivanovSPvirtru) [reply lib/tdf3/src/models/attribute-set.ts:1]: Test running in the browser showed that it had node js dependency that was not recognized by browser. So ajv is browser first lib

File: lib/tdf3/src/version.ts (hunk 3 above)
Comment (dmihalcik-virtru) [reply lib/tdf3/src/version.ts:1]: It is the package version. See: https://github.com/opentdf/client-web/blob/main/scripts/bump-version.sh#L49

**Review by mustyantsev — APPROVED** [review]

**Review by sievdokymov-virtru — APPROVED** [review]

File: lib/tdf3/src/FileClient.ts [hunk 4]
```diff
@@ -68,7 +66,7 @@ export class FileClient {
       let url;
       try {
         url = new URL(source);
-        if (!['http', 'https'].includes(url.protocol)) {
+        if (!['http:', 'https:'].includes(url.protocol)) {
```
Comment (dmihalcik-virtru) [inline lib/tdf3/src/FileClient.ts:69]: !!

**Review by dmihalcik-virtru — APPROVED** [review]

**Pull request merged**